wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -851,6 +851,111 @@ <h2>Icons</h2>
 
         </section>
 
+        <section id="icons" class="icons">
+            <h2>SVG Icons</h2>
+
+          <style>
+            .spinning {
+              animation-name: spin;
+              animation-duration: 500ms;
+              animation-iteration-count: infinite;
+              animation-timing-function: linear;
+            }
+            @keyframes spin {
+                from {
+                    transform: translateY(0.6em) rotate(0deg);
+                } to {
+                    transform: translateY(0.6em) rotate(360deg);
+                }
+            }
+
+          </style>
+
+            <ul class="unlist">
+                <li>{% icon 'wagtail-icon' %} wagtail</li> {# Note: There is 'wagtail' id on the body named 'wagtail'. Should prefix (namespace) all icons? #}
```

Comment:
Ah, that note was a good catch, but it’s now outdated.

Reply:
Hah, yes! Will remove it.

Reply:
Removed